### PR TITLE
Update (roll back) version of ESMF used by chgres_cube on hera.  

### PR DESCRIPTION
**Pull request by gsketefian**
## Description of Changes
This is necessary because the previously used version of ESMF (8.0.0bs21) was modified (by the hera admins or EMC) so that it breaks the regional_workflow (the make_ics task fails).  Thus, we roll back the version of ESMF to something slightly earlier (8.0.0).

## Tests Conducted
Built chgres_cube with the updated version on hera and ran the workflow (all WE2E tests).  It now works.

## Other Notes
Once this is accepted, the hash # in the Externals.cfg file of the ufs-srweather-app (master branch), stanza [ufs_utils_chgres] needs to be updated to the new hash created by this PR.

### CODE REVIEW

**Comment by jwolff-ncar** [comment]
I passed this modification on to Lindsay yesterday to test after she ran into trouble and her build was successful on hera after this change.

**Comment by gsketefian** [comment]
> I passed this modification on to Lindsay yesterday to test after she ran into trouble and her build was successful on hera after this change.

Good!  I'm waiting for at least one approval to merge this in.  Others testing their PRs in regional_workflow will fail if this doesn't go in.  So let's get it in ASAP!

**Review by JeffBeck-NOAA — APPROVED** [review]

**Pull request merged**
